CAFAna/Analysis/CMakeLists.txt

```diff
@@ -25,17 +25,15 @@ set(Analysis_header_files
   CheckPointHelper.h)
 
 add_library(CAFAnaAnalysis SHARED ${Analysis_implementation_files})
-target_link_libraries(CAFAnaAnalysis ${CAFANACOREEXT})
```

Comment:
Do I detect the hand of Luke in all the makefile changes? Do we promise that the state is better than it was before? (I know so little about cmake I have absolutely no way to tell...)

Reply:
Yes Luke has worked his cmake magic. I have no idea what the changes do though. Could be worth asking Luke.